### PR TITLE
Build: Use the US spelling of "favor"

### PR DESCRIPTION
**Pull request by Necmttn**
For reference: https://en.wikipedia.org/wiki/Wikipedia:Lists_of_common_misspellings/For_machines

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -308,7 +308,7 @@ Martijn W. van der Lee <martijn@vanderlee.com>
 Devin Wilson <dwilson6.github@gmail.com>
 Damian Senn <jquery@topaxi.codes>
 Zack Hall <zackhall@outlook.com>
-Vitaliy Terziev <vitaliyterziev@gmail.com>
+Vitality Terziev <vitaliyterziev@gmail.com>
```
Comment (mgol) [inline AUTHORS.txt:311]: That doesn't look intended.

File: test/data/jquery-1.9.1.js [hunk 2]
```diff
@@ -4328,7 +4328,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 		assert( function( div ) {
 
 			// Select is set to empty string on purpose
-			// This is to test IE's treatment of not explictly
+			// This is to test IE's treatment of not explicitly
```
Comment (mgol) [inline test/data/jquery-1.9.1.js:4331]: This is jQuery 1.9.1; it's fixed in time and should never be changed.

File: test/index.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
 			amd = QUnit.config.amd || QUnit.urlParams.amd;
 
 		// Workaround: Remove call to `window.__karma__.loaded()`
-		// in favour of calling `window.__karma__.start()` from `loadTests()`
```
Comment (mgol) [inline test/index.html:38]: It's not incorrect, this is just British spelling. I think in principle we follow American English, though, so it looks like a good change.

**Comment by Necmttn** [comment]
Hey @mgol, just reverted the changes you pointed. 
thank you for checking.

File: test/unit/selector.js [hunk 4]
```diff
@@ -2125,7 +2125,7 @@ QUnit.test( "jQuery.escapeSelector", function( assert ) {
 	assert.equal( jQuery.escapeSelector( "\x20\x21\x78\x79" ), "\\ \\!xy",
 		"Escapes non-word ASCII characters" );
 
-	// Astral symbol (U+1D306 TETRAGRAM FOR CENTRE)
```
Comment (mgol) [inline test/unit/selector.js:2128]: This is the official name for the character so it shouldn't be changed: https://www.fileformat.info/info/unicode/char/1d306/index.htm

**Comment by mgol** [comment]
Landed, thanks!

**Pull request merged**